From: "Ben Dooks (Codethink)" <ben.dooks@codethink.co.uk>
Date: Fri, 18 Oct 2019 20:20:24 -0700
Subject: mm/init-mm.c: include <linux/mman.h> for vm_committed_as_batch
Patch-mainline: v5.4-rc4
Git-commit: a2ae8c0551050cbf3232fbb3d963a9bbe7b57268
References: jsc#SLE-16407

mm_init.c needs to include <linux/mman.h> for the definition of
vm_committed_as_batch.  Fixes the following sparse warning:

  mm/mm_init.c:141:5: warning: symbol 'vm_committed_as_batch' was not declared. Should it be static?

Link: http://lkml.kernel.org/r/20191016091509.26708-1-ben.dooks@codethink.co.uk
Signed-off-by: Ben Dooks <ben.dooks@codethink.co.uk>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 mm/init-mm.c |    1 +
 1 file changed, 1 insertion(+)

--- a/mm/init-mm.c
+++ b/mm/init-mm.c
@@ -5,6 +5,7 @@
 #include <linux/spinlock.h>
 #include <linux/list.h>
 #include <linux/cpumask.h>
+#include <linux/mman.h>
 
 #include <linux/atomic.h>
 #include <linux/user_namespace.h>
